<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>workCompare</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow
{
border-image: url(:/res/background.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>211</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>211</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>2.打开班级已交的作业</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>551</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>551</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>211</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>211</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>1.打开存取班级姓名的文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 已交的同学：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string> 未交的同学：</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
